driver/apdu/at.c

```diff
@@ -74,13 +75,15 @@ static int apdu_interface_connect(struct euicc_ctx *ctx)
         fprintf(stderr, "Device missing AT+CCHC support\n");
         return -1;
     }
+    /* AT+CGLA=? might not be available but AT+CGLA= is
+       Sierra wireless MC7455 supports both but EM7590 only the latter
     fprintf(fuart, "AT+CGLA=?\r\n");
     if (at_expect(NULL, NULL))
     {
         fprintf(stderr, "Device missing AT+CGLA support\n");
         return -1;
     }
-
+    */
```

Comment:
I don't think it is that important to be honest, it will just fail after the first command, checking for AT+CGLA= requires to actually do AT+CCHO and then AT+CGLA so what are we gaining here.

Maybe change it to a warning? and again the issue is the lack of AT+CGLA= and not AT+CGLA=? where in AT commands those are separated commands so just like in the case of sierra wireless it might not be implemented